source/languages/en/dataplatform/using-dataplatform/configuration/getting-started-with-cache-proxy.md

```diff
@@ -221,7 +221,7 @@ telnet $CACHE_PROXY_HOST $CACHE_PROXY_STATISTICS_PORT
 
 Returns statistic results:
 
-```javascript
+```Json
```

Comment:
Agreed. @ph07 you can update the PR with this change, or I can close this PR and make the change myself. Just let me know.